src/intents/DSL/schema.ts

```diff
@@ -55,12 +60,39 @@ export const nonceConditionSchema = joi.object({
 
 const priceOperators = Object.values(PriceOperator)
 
+const toTokenWithDecimals = (value: any, helpers: any) => {
+  const chainId = helpers.prefs.context.chainId;
+
+  try {
```

Comment:
remove the try/catch unless there's a specific need for it

is there some expected error condition from the `toTokenWithDecimalsArgs` call that you wanted to handle here?

Reply:
yeah sorry it was for debugging